config/config.go

```diff
@@ -176,3 +180,14 @@ func (c *Config) ConfigurePlugins(ctx context.Context, path string) (map[string]
 	}
 	return sources, destinations, nil
 }
+
+func firstPath(paths []string, filename string) (string, error) {
+	for _, p := range paths {
+		fpath := filepath.Join(p, filename)
+		_, err := os.Stat(fpath)
+		if err == nil {
+			return fpath, nil
+		}
+	}
+	return "", os.ErrNotExist
```

Comment:
do we maybe care about the _real_ reason for the error? Like, what if the `Stat` operation failed for a different reason?

Reply:
ok we will only fall back to `os.ErrNotExist`